app/ts/utils/errors.ts

```diff
@@ -39,3 +40,13 @@ export function printError(error: unknown) {
 	}
 	return console.error(error)
 }
+
+export async function handleUnexpectedError(error: unknown) {
+	printError(error)
+	await sendPopupMessageToOpenWindows({
+		method: 'popup_UnexpectedErrorOccured' as const,
+		data: {
+			message: typeof error === 'object' && error !== null && 'message' in error && error.message !== undefined && typeof error.message === 'string' ? error.message : 'Please see The Interceptors console for more details on the error.'
```

Comment:
"Interceptor's console" feels like an actual UI within interceptor, just a few reword.

```suggestion
			message: typeof error === 'object' && error !== null && 'message' in error && error.message !== undefined && typeof error.message === 'string' ? error.message : 'Check the console for The Interceptor logs to get more details on the error.'
```